src/OSPSuite.UI/Services/DiagramZoomRectangleService.cs

```diff
@@ -107,25 +107,10 @@ private bool canZoom()
          return !_selectionRectangle.IsEmpty && _selectionRectangle.Height > 3 && _selectionRectangle.Width > 3;
       }
 
-      private bool mouseIsOverChart(MouseEventArgs e)
+      private bool mouseIsOverDiagram(MouseEventArgs e)
       {
-         var pointLocation = pointLocationAt(e);
-         return pointLocation != null && !pointLocation.IsEmpty;
-      }
-
-      private DiagramCoordinates pointLocationAt(MouseEventArgs e)
-      {
-         if (_chartControl.XYDiagram == null)
-            return null;
-
-         try
-         {
-            return _chartControl.XYDiagram.PointToDiagram(new Point(e.X, e.Y));
```

Comment:
This will return `pointLocation.IsEmpty = true` when the point is not in the diagram